### ISSUE TITLE
1.8.0 release notes document undefined option `--dangling`

### ISSUE THREAD

**Issue by albers** [docker/docker#15518]
In https://github.com/docker/docker/releases/tag/v1.8.0, section "Bug Fixes":

>Make `docker rmi --dangling` safe when pulling

```shell
$ docker rmi --dangling
flag provided but not defined: --dangling
See 'docker rmi --help'.
```

**Comment by albers** [comment]
It's also in CHANGELOG.md.

Probably refers to deleting dangling images that were identified with the dangling filter of docker images:
docker rmi $(docker images -q -f dangling=true)

Probably refers to #13681

**Comment by albers** [comment]
#15519 addresses the changelog part.

**Comment by cpuguy83** [comment]
Thanks!

**Comment by albers** [comment]
@thaJeztah same as in #15520: As the release notes were not updated, this issue should not be closed.

**Comment by thaJeztah** [comment]
Right, GitHub did that automatically ;-)

ping @calavera for updating the release notes accordingly :smile:

**Comment by calavera** [comment]
Release notes updated. Closing this issue. Thank you!

**Comment by albers** [comment]
Thanks @calavera. Could you please also update the release notes according to #15520? This issue was automatically closed when the changelog part (#15521) was merged. It should still be open.
